monai/run_inference.py

```diff
@@ -0,0 +1,317 @@
+"""
+Script to run inference on a MONAI-based model for contrast-agnostic soft segmentation of the spinal cord.
+Predictions are stored in independent folders for each subject. Summary of the test metrics (both per subject and overall)
+are stored in a json file, along with the time taken for inference.
+
+Usage:
```

Comment:
usage should not be duplicated from the get parser function— eg if someone updates the parser but forgets to update this section it creates confusion— better to redirect and avoid duplication

Reply:
Ah I never realized that `-h` also gives a usage example. Thanks for pointing this out! 
Updated in https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/pull/60/commits/2a18878d4d11509a2a08094960e81ea50b4bc510